src/pages/CheckoutStepOne.page.js

```diff
@@ -0,0 +1,25 @@
+// @ts-check
+import { BaseSwagLabPage } from './BaseSwagLab.page';
+
+export class CheckoutStepOnePage extends BaseSwagLabPage {
+    url = '/checkout-step-one.html';
+
+    checkoutFirstName = this.page.locator('[id="first-name"]');
+
+    checkoutLastName = this.page.locator('[id="last-name"]');
+
+    checkoutZip = this.page.locator('[id="postal-code"]');
+
+    checkoutContinueButton = this.page.locator('[id="continue"]');
+
+    /**
+     * @param {string} firstName
+     * @param {string} lastName
+     * @param {string} zip
+     */
+    async fillYourInformation(firstName, lastName, zip) {
```

Comment:
Add a check that after filling in the fields, the continue button becomes active, or vice versa, if any field is empty, an error appears